src/test/shared/filesystem.test.ts

```diff
@@ -1,62 +0,0 @@
-/*!
- * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0
- */
-
-import * as assert from 'assert'
-import * as del from 'del'
-import * as fs from 'fs'
-import * as path from 'path'
-import { CustomPromisify, promisify } from 'util'
-
-import * as filesystem from '../../shared/filesystem'
-import { fileExists, makeTemporaryToolkitFolder } from '../../shared/filesystemUtilities'
-import { getPropAs } from '../../shared/utilities/tsUtils'
-
-const functionsToTest = ['access', 'readFile', 'readdir', 'rename', 'stat', 'mkdtemp', 'unlink', 'writeFile']
-
-describe('filesystem', () => {
-    functionsToTest.forEach((fxName: string) => {
-        it(`filesystem.${fxName} is same as promisify(fs.${fxName})`, async () => {
-            const filesystemFunction = getPropAs<Function>(filesystem, fxName) // filesystem[fxName]
-            const fsFunction = getPropAs<CustomPromisify<Function>>(fs, fxName) // fs[fxName]
-            const actualType = typeof filesystemFunction
-            assert(actualType === 'function', `filesystem.${fxName} should be a "function" but is "${actualType}"`)
-            assert.strictEqual(String(filesystemFunction), String(promisify(fsFunction)))
-        })
-    })
-
-    describe('mkdir', async () => {
```

Comment:
`mkdir` is still a custom implementation. Seems like we'd still want the mkdir related tests.

Reply:
see comment above, this is removed here to pass tests and fixed and re-added by the other PR

Reply:
This is the wrong file, oops, re-adding test